app/lib/actions/try.rb

```diff
@@ -0,0 +1,51 @@
+module Actions
+  module Try
+    # A set of actions to try different scenarios with tasks in development.
+    #
+    # Usage:
+    # from rails console, run:
+    #
+    #    # Successful task (ending in stopped state/success result)
+    #    ForemanTasks.sync_task(Actions::Try::Success)
+    #
+    #    # Failing task task (ending in paused state/error result)
+    #    ForemanTasks.sync_task(Actions::Try::Pause)
+    #
+    #    # Task that fails but resumes (ending in stopped state/warning result)
+    #    ForemanTasks.sync_task(Actions::Try::Skip)
+    unless Rails.env.production?
```

Comment:
I'd move this one level higher so we won't end up with empty `Actions::Try` module in production environments